parking_permits/models/order.py

```diff
@@ -237,10 +237,14 @@ def create_renewal_order(
         Create new order for updated permits information that affect
         permit prices, e.g. change address or change vehicle.
         """
-        customer_permits = ParkingPermit.objects.filter(customer=customer)
+        customer_permits = ParkingPermit.objects.filter(
+            customer=customer, status=ParkingPermitStatus.VALID
+        )
```

Comment:
Does `_validate_customer_permits()` loop through `customer_permits`?

If so, then this line 

```python
first_permit = customer_permits.first()
if not first_permit:
```

could be replaced by: 

```
try:
    first_permit = customer_permits[0]
except IndexError:
    # raise error etc
```

This would then skip the extra database query.

Another thing is that it's not clear what the order of the permits are in e.g. creation date. If not defined by the `ordering` in the model, this should be explicit.